### PR TITLE
Add vagrant support for 0.9.54 testing

### PR DESCRIPTION
**Pull request by barnabycourt**
Back port the vagrant support from master

### CODE REVIEW

**Comment by cnsnyder** [comment]
Will look at this tomorrow (or as soon as I'm done upgrading os verisons)

**Comment by Ceiu** [comment]
@candlepin/jenkins-bot retest this please

File: vagrant/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+* NFS mount your local source directory into the VM
+* Configure Tomcat remote debugging 
+* Install Postgres and configure the candlepin user
+  without an external messaging broker).
+* Compile & install Candlepin
+* Forward Ports to your local system for debugging
+  * Tomcat remote debug on port 8000
+  * REST access for Candlepin on port 8080 and 8443
+
+## Prerequisits
```
Comment (cnsnyder) [inline vagrant/README.md]: *Prerequisites

**Comment by cnsnyder** [comment]
Will merge when tests complete.

**Pull request merged**